addons/website_event/models/website_menu.py

```diff
@@ -19,6 +19,7 @@ def unlink(self):
                     to_update.append(fname)
 
         # call super that resumes the unlink of menus entries (including website event menus)
+        website_event_menus.unlink()
```

Comment:
Isn't it automatic with:
`menu_id = fields.Many2one('website.menu', string='Menu', ondelete='cascade')` 

?

Reply:
Well issue is that it is unlink at db level, meaning ``unlink`` of website_event_menu models is not called, and view is therefore not removed. We have to know that ondelete='cascade' is done at db level, not model level. Logic has to be manually called.